ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -260,7 +260,13 @@ fi
 navigator start
 
 # Try to start the microdds_client with UDP transport if module exists
-microdds_client start -t udp -p 8888
+if [ $px4_instance -eq 0 ]
+then
+	microdds_client start -t udp -p 8888
+else
+	param set XRCE_DDS_KEY ${px4_instance}
+	microdds_client start -t udp -p 8888 -n "px4_$px4_instance"
+fi
```

Comment:
I suggest using an environment variable for the namespace, e.g., "PX4_MICRODDS_NS". This gives the user more freedom of selecting the namespace, e.g. based on vehicle. Then, the rcS would look something like this 
if [ ! -z $PX4_MICRODDS_NS ] ; then
param set XRCE_DDS_KEY ${px4_instance}
microdds_client start -t udp -p 8888 -n ${PX4_MICRODDS_NS} 
else 
microdds_client start -t udp -p 8888

What do you think of this?